vosekast_control/TestSequence.py

```diff
@@ -95,18 +96,28 @@ def __init__(
 
     async def write_loop(self):
         try:
+            # get start time
             time_sequence_t0 = datetime.now()
             delta_time_sequence = 0
-            # null scale
+
+            # tare scale
             if abs(self.scale.scale_history[0]) < 0.15:
                 scale_nulled = self.scale.scale_history[0]
             else:
                 scale_nulled = 0
 
+            # generate sequence_id
+            sequence_id = random.randint(10000000000, 100000000000)
+
+            # connect db
+            DBConnection.connect()
```

Comment:
Why is the connect function here? I think the better place is the main function. 
Usually you try to hold function that open and close connections close together.

Reply:
moved to main